lib/APPTurtle/src/App.cpp

```diff
@@ -274,10 +274,9 @@ bool App::setupSerialMuxProtServer()
     bool isSuccessful = false;
 
     m_serialMuxProtChannelIdStatus = m_smpServer.createChannel(STATUS_CHANNEL_NAME, STATUS_CHANNEL_DLC);
-    m_serialMuxProtChannelIdMotorSpeeds =
-        m_smpServer.createChannel(SPEED_SETPOINT_CHANNEL_NAME, SPEED_SETPOINT_CHANNEL_DLC);
+    m_serialMuxProtChannelIdTurtle = m_smpServer.createChannel(TURTLE_CHANNEL_NAME, TURTLE_CHANNEL_DLC);
 
-    if ((0U == m_serialMuxProtChannelIdStatus) || (0U == m_serialMuxProtChannelIdMotorSpeeds))
+    if ((0U == m_serialMuxProtChannelIdStatus) || (0U == m_serialMuxProtChannelIdTurtle))
     {
         LOG_ERROR("Failed to create SerialMuxProt channels.");
     }
```

Comment:
Isn't this fatal ? I wonder why this is just a log but no SW reaction. What happens later on this error happend ?
Shouldn't we go into a HALT with error indication (like RED LED flash and RU reset pin trigger).

Reply:
It is in fact fatal. Please see line 136